app/assets/stylesheets/application.scss

```diff
@@ -8,7 +8,9 @@ $govuk-new-link-styles: true;
 $govuk-include-default-font-face: false;
 
 @import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/components/intervention";
 
+@import "components/intervention";
```

Comment:
Is this line needed? Not sure it is - I think that would be if there was a component called intervention defined in this app.

Reply:
Ahhh sorry yep thanks - I see!